OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs

```diff
@@ -20,35 +23,42 @@ public class CodeficatorAddressDto
 
     public double Longitude { get; set; }
 
+    public int Order { get; set; }
+
     public string FullName
     {
         get
         {
-            string addr = CityDistrict;
+            StringBuilder addr = new StringBuilder();
```

Comment:
Please set a clear name for the variable

Reply:
done